CIP-0067/README.md

```diff
@@ -0,0 +1,53 @@
+---
+CIP:
+Title: Asset Name Label Registry
+Authors: Alessandro Konrad <alessandro.konrad@live.de>, Thomas Vellekoop <thomas.vellekoop@iohk.io>
+Comments-URI:
+Status: Draft
+Type: Informational
+Created: 2022-07-13
+Post-History:
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This proposal defines a standard to classify Cardano native assets by the asset name.
+
+## Motivation
+
+As more assets are minted and different standards emerge to query data for these assets, it's getting harder for 3rd parties to determine the asset type and how to proceed with it. This standard is similar to [CIP-0010](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0010), but focuses on the asset name of an asset.
+
+
+## Specification
+
+To classify assets the `asset_name` needs to be prefixed with an opening and closing parentheses and the label in between: `({Label})`.
```

Comment:
Also, additional suggestion: whatever the choice of format it, perhaps give an [ABNF](https://en.wikipedia.org/wiki/Augmented_Backus–Naur_form) syntax to describe it so that symbols aren't misinterpreted (one person reading `({label})` might think for its label 42, the prefix would be: `({42})`. 

```abnf
asset-name = asset-name-label asset-name-body

asset-name-label = "(" 1-5DIGIT ")"

asset-name-body = *OCTET ; exact length depends on the asset-name-label's length
```

Reply:
*I am only seeing / reading this comment now: https://github.com/cardano-foundation/CIPs/pull/298#issuecomment-1233421621